src/documents/IoCAPI.md

```diff
@@ -145,3 +161,193 @@ component member name.
+                concludeComponentObservation: {
+                    funcName: "fluid.concludeComponentObservation",
+                    priority: "first"
+                },
+                concludeComponentInit: {
+                    funcName: "fluid.concludeComponentInit",
+                    priority: "last"
+                }
+            }
+        }
+    });
+```
+
+This shows the configuration of two [local workflows](#local-workflows), `fluid.concludeComponentObservation` which will execute
+first, and `fluid.concludeComponentInit` which will execute last. The former completes the process of evaluating all component
+options, members, invokers and other top-level masterial, and the latter marks the component as fully constructed and fires its
+`onCreate` event.
+
+The definitions from `fluid.modelComponent` look as follows:
+
+```javascript
+    fluid.defaults("fluid.modelComponent", {
+        gradeNames: ["fluid.component"],
+
+        workflows: {
+            global: {
+                resolveModelSkeleton: {
+                    funcName: "fluid.resolveModelSkeleton"
+                }
+            },
+            local: {
+                notifyInitModel: {
+                    funcName: "fluid.notifyInitModel",
+                    priority: "before:concludeComponentInit"
+                }
+            }
+        }
+// ...
+    });
+```
+
+This shows the registration of one global workflow, `fluid.resolveModelSkeleton` which since it is the only one in the core
+framework has no priority annotation. It also shows the registration of a local workflow, `fluid.notifyInitModel` which
+is responsible for firing any [model listeners](ChangeApplierAPI.md#declarative-style-for-listening-to-changes) which need
+to be notified of initial model values as part of the [initial transaction](ModelRelay.md#the-initial-transaction). The
+[priority|Priorities.md] field attached here indicates that this should happen (if no further definition intervenes) immediately
+before the `concludeComponentInit` action which fires `onCreate`.
+
+### Global workflows
+
+There are two kinds of workflow functions which may be contributed, global and local workflows. Global workflows represent
+particularly ambitious functionality which requires oversight of the entire instantiating tree, and all global workflows
+execute before any local workflows. The framework currently includes just one global workflow, the one which identifies and resolves the values
+of all model components throughout the instantiating tree's `model skeleton`, but further ones will be implemented as part of the upcoming
+framework's markup renderer for [FLUID-4260](https://issues.fluidproject.org/browse/FLUID-4260).
+
+A global workflow function receives the signature:
+
+### &lt;global.workflow.function.name&gt;(shadows)
+
+* `shadows {Array of Shadow}` An array of _all_ component shadows participating in this transaction, sorted in order from the root of the component
```

Comment:
I realize they may not be new, as they appear briefly in the doco for fluid.globalInstantiator, but since they are part of these method arguments, could we maybe have a short paragraph somewhere on this page, describing what a 'shadow' record is.

Reply:
It does describe it in the next sentence, at least about as much as it needs to be here ...